axiom/nr_attributes.c

```diff
@@ -97,6 +97,15 @@ void nr_attribute_config_disable_destinations(nr_attribute_config_t* config,
   config->disabled_destinations |= disabled_destinations;
 }
 
+void nr_attribute_config_enable_destinations(nr_attribute_config_t* config,
+                                             uint32_t disabled_destinations) {
```

Comment:
This argument name is confusing to me. Shouldn't it be called enabled_destinations?

Reply:
That would make it clearer - thanks for the suggestion.